src/genie/libs/parser/nxos/show_fex.py

```diff
@@ -0,0 +1,61 @@
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any
+
+# Python
+import re
+
+# ==============================
+# Schema for 'show fex'
+# ==============================
+
+class ShowFexSchema(MetaParser):
+
+    ''' Schema for "show fex" '''
+
+    schema = {
+        'fexes': {
+            Any(): {
+                'description': str,
+                'state': str,
+                'model': str,
+                'serial_number': str
+            }
+        }
+    }
+
+# ==============================
+# Parser for 'show fex'
+# ==============================
+
+class ShowFex(ShowFexSchema):
+
+    ''' Parser for "show fex"'''
+
+    cli_command = 'show fex'
+
+    def cli(self, output=None):
+        if output is None:
+            # Get output from device
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output 
+        # Initial return dictionary
+        parsed_dict = {}
+
+        p1 = re.compile(r'(?P<number>(\d+)) +(?P<description>(\S+)) +(?P<state>(\S+)) +(?P<model>(\S+)) +(?P<serial>(\S+))$')
+
+        for line in out.splitlines():
+            line = line.strip()
+            m = p1.match(line)
```

Comment:
same thing about comment showing what the regex matches

Reply:
Same as above, comments were added in 5261d03.